mobile.go

```diff
@@ -104,7 +105,10 @@ func doUpdateMobile(url string, out *os.File, updater Updater) error {
 	}
 
 	req.Header.Add("Accept-Encoding", "gzip")
-	pro.PrepareForFronting(req)
+
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
```

Comment:
This is also very cool @atavism, and I just realized we actually don't even domain front to updates on desktop at all. Were you able to verify the front works here too? I guess it's theoretically no different from `api.getiantem.org` or the config stuff, but curious is were able to reproduce the fronting version definitely working end to end. I should definitely add this to the desktop version if so.

On the desktop side we also config this slightly differently just via the passed in http client in the config here:

https://github.com/getlantern/flashlight/blob/devel/autoupdate/autoupdate.go#L52

and here:

https://github.com/getlantern/flashlight/blob/devel/autoupdate/autoupdate.go#L89